airlock/settings.py

```diff
@@ -63,6 +63,15 @@ def get_env_var(name):
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = get_env_var("DJANGO_DEBUG") == "True"
 
+if DEBUG:  # pragma: no cover
+    DEBUG_TOOLBAR = os.environ.get("DEBUG_TOOLBAR", "false").lower() in [
```

Comment:
For consistency with existing env vars (`DJANGO_DEBUG`, `DJANGO_ALLOWED_HOSTS`) the env var here should probably be `DJANGO_DEBUG_TOOLBAR`. Although admittedly this is a dev only setting gated behind `DJANGO_DEBUG` so I feel a bit less strongly about the namespacing here.